package.json

```diff
@@ -11,8 +11,8 @@
     }
   ],
   "scripts": {
-    "test": "./scripts/run_tests.sh",
-    "prodTest": "npm run buildProdBrowser && sed 's/var rudderanalytics/rudderanalytics/g' dist/rudder-analytics.min.js > __tests__/prod_test.js && jest script.test.js",
+    "test": "jest --coverage",
```

Comment:
Can we add `--notify` as well? Good to have!

Reply:
okay